### PR TITLE
grammar, wording, link fixes + one note

### PR DESCRIPTION
**Pull request by gramsimamsi**
tried to leave the meaning untouched and only corrected formality-related things, with one exception:
I added a possible citation for threat 3, orchestration tool: https://redlock.io/blog/cryptojacking-tesla

### CODE REVIEW

**Comment by drwetter** [comment]
Hi'doin' Lukas,

thanks!

I may not always have used a grammar which is 100% in line with how it's supposed to be. However I would feel more comfortable if a native English speaker will review this PR or what I wrote in general. (I remember, you're German, too.)

Anybody out there?

Cheers, Dirk

PS: Sorry not getting back to your mail

**Comment by hartwork** [comment]
@gramsimamsi could you rebase this against latest master to get rid of the conflicts?

**Comment by drwetter** [comment]
@hartwork  see my comment

File: README.md [hunk 1]
```diff
@@ -4,37 +4,37 @@ This is the OWASP Docker Top 10. It's a work in progress.
...
+These 10 bullet points are mostly (see below this paragraph) about system and network security and system and network architecture. As a developer you don't have to be an expert in those - that's what this guide is for. But as indicated above, the best point to start thinking about and addressing those topics is early on. Please do not just start building it.
 
-One of the bullet point should not be misunderstood: Patch management is not a techincal point. It's a management process. Last but not least for technical or information security management who has not been much worried about containerization this document also provides insights about the risks involve.
+One of the bullet point should not be misunderstood: Patch management is not a techincal point, it's a management process. Last but not least,  this document also provides insights about the risks involved. This document might prove useful to technical or information security management people, who have not worried a lot about containerization so far.
```
Comment (hartwork) [inline README.md:21]: Typo "techincal" remains unfixed

File: 000 - Introduction.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
 
 # Introduction
 
-Implementing a containerized environment is changing a lot not only how deployments are done but it also has a huge impact on a system and networking level and how hardware and network resources are being used.
+Implementing a containerized environment changes a lot not. It doesn't only affect deployments and the way they are done, but also has a huge impact on both system and networking levels, as well as hardware and network resource usage.
```
Comment (hartwork) [inline 000 - Introduction.md:4]: The part with `changes a lot not.` looks mistaken to me. Did you forget to drop `not` here?

**Comment by hartwork** [comment]
> see my comment

Regarding German as a native language? That's a fair point.

**Comment by drwetter** [comment]
@hartwork : 

1) For the content there will be changes >= late November. So a large review of the grammar doesn't make much sense atm. Later on a native English speaker needs to take this on.

2) For smaller and OBVIOUS errors and for the time being: they can go into separate PRs. Obvious error are just typos or sentences where e.g. a word is missing or a wrong word is being used. I do not discuss punctuation or sentences which aren't perfect yet. This will be a task of a native **English** speaker.

**Pull request closed without merging**